### PR TITLE
Changes to api-stats-updates.md

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
Changed file name and type, and converted content to asciidoc. Minor text changes.

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+//
+// In the title, include nouns that are used in the body text. This helps
+// readers and search engines find information quickly.
+[id="monitoring-container-health-and-api-status_{context}"]
+= Monitoring container health and API status
+
+Container health assesses the status of each of the {OpenStack} service containers
+by periodically running a health check script using the docker standard.
+Most {OpenStack} services implement a healthcheck that logs issues and returns a
+binary status, healthy or unhealthy. For the {OpenStack} APIs, the healthchecks
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc:29]:
```suggestion
binary status. For the {OpenStack} APIs, the healthchecks
```

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* You are using the most current version of {OpenStack} {OpenStackVersion}.
+
+.Procedure
+
+. Open the `stf-connectors.yaml` and edit the `collectd-sensubility` paramater:
++
+----
+CollectdSensubilityTransport: amqp1
+----
+. Optional: In a multicloud environment, you must configure `collectd-sensubility` events channel with the new collectd events address. Edit the `stf-connectors.yaml` file:
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc:47]:
```suggestion
. If you have a multicloud environment, you must configure `collectd-sensubility` events channel with the new collectd events address. Edit the `stf-connectors.yaml` file:
```

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// In the title, include nouns that are used in the body text. This helps
+// readers and search engines find information quickly.
+[id="monitoring-container-health-and-api-status_{context}"]
+= Monitoring container health and API status
+
+Container health assesses the status of each of the {OpenStack} service containers
+by periodically running a health check script using the docker standard.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc:27]:
```suggestion
by periodically running a health check script using the Docker standard.
```

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc:27]: Should Docker be capitalized? Also we're using `podman` in 16.1 so I think the more correct term is OCI? (Open Container Initiative)

I'm not actually sure what the correct term is here. @csibbitt do you know?

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_monitoring-container-health-and-api-status.adoc:27]: (Also sorry, I forgot this was a mild change to Pauls root PR and is probably where this comment should have landed.)

**Review by leifmadsen — APPROVED** [review]

**Pull request merged**